### ISSUE TITLE
PATCH method errors on requestSignature

### ISSUE THREAD

**Issue by BerniWittmann** [wultra/powerauth-mobile-sdk#601]
I figured out that when calling `requestSignature` the HTTP method `PATCH` does not seem to be supported as it results in a Signature Error. 

Example: 

```
const auth = PowerAuthAuthentication.password("1234");
const signature = await powerAuth.requestSignature(auth, "PATCH", "/api/endpoint", "{ \"myJson\": \"body\" }")

```

The above throws an error, but switching it out for "PUT" or "POST" works perfectly fine. Note: I tested this on the ios simulator.

Luckily this is not a blocking issue for us right now, but still wanted to let you know

**Comment by BerniWittmann** [comment]
Thank you :) 

Yes no issue for us, as we switched to PUT now, but just wanted to let you know :)

**Comment by hvge** [comment]
Hi @BerniWittmann, thank you for the report. I have already created a fix that will be released with the next SDK update in the 1.8.x release branch. If it’s important to you, I can release it sooner, but I would prefer to wait and resolve more issues in the update.

**Comment by BerniWittmann** [comment]
Hi @hvge Wow that was fast :) Please take all the time you need, no urgency required from my side
